backend/tracim_backend/lib/utils/utils.py

```diff
@@ -228,12 +229,16 @@ def string_to_list(
     :param base_string: entry string which should be converted.
     :param separator: string separator,
     :param cast_func: all item should be casted to this function, this help
+    :param stripped: strip string (remove beginning and ending whitespace) of
```

Comment:
if the parameter is a boolean, then the explanation must be : "if True xxx else xxx". Your explanation is not clear here, that if true then it will strip otherwise it wont

Reply:
done.